wheatley/bot.py

```diff
@@ -279,18 +283,24 @@ def tick(self) -> None:
                 self._should_stand = False
                 self._is_ringing = False
 
+            # ===== CONVERT THE FLAGS INTO ACTIONS =====
+
+            # If the `_should_start_method` flag is set and Wheatley is:
+            # - ringing rounds
+            # - on the row generator's start stroke
+            # then Wheatley should go into changes
```

Comment:
This describes what the if statement below checks rather than why it checks it which is where I feel comments add value. It's also easy for a later change to make the comment and check become different

Reply:
Ah yes good point - I'll just remove the comment.